aviary/utils/process_input_decks.py

```diff
@@ -60,18 +60,30 @@ def create_vehicle(vehicle_deck='', verbosity=Verbosity.BRIEF, meta_data=_MetaDa
     aircraft_values.set_val(Settings.PROBLEM_TYPE, val=ProblemType.SIZING)
     aircraft_values.set_val(Aircraft.Electrical.HAS_HYBRID_SYSTEM, val=False)
 
+    initial_guesses = {
+        # initial_guesses is a dictionary that contains values used to initialize the trajectory
+        'actual_takeoff_mass': 0,
+        'rotation_mass': 0,
+        'operating_empty_mass': 0,
+        'fuel_burn_per_passenger_mile': 0,
+        'cruise_mass_final': 0,
+        'flight_duration': 0,
+        'time_to_climb': 0,
+        'climb_range': 0,
+        'reserves': 0
+    }
+
     if isinstance(vehicle_deck, AviaryValues):
         aircraft_values.update(vehicle_deck)
-        initial_guesses = {}
     else:
         vehicle_deck = get_path(vehicle_deck)
         aircraft_values, initial_guesses = parse_inputs(
-            vehicle_deck, aircraft_values, meta_data=meta_data)
+            vehicle_deck, aircraft_values, initial_guesses, meta_data=meta_data)
```

Comment:
now that this function call is getting more complicated could we add variable names? Ie. `parse_inputs(vehicle_deck=vehicle_deck, aircraft_values=aircraft_values`...?

Reply:
I've updated this as suggested.